app/javascripts/components/playSlot/game.jsx

```diff
@@ -59,54 +59,68 @@ const LINE_CASES = [
   [{ symbol: 4, length: 3 }, { symbol: 3, length: 4 }, { symbol: 2, length: 5 }], // 75
   [{ symbol: 3, length: 3 }, { symbol: 2, length: 4 }, { symbol: 1, length: 5 }], // 50
   [{ symbol: 2, length: 3 }, { symbol: 1, length: 4 }, { symbol: 0, length: 5 }], // 25
-  [{ symbol: 0, length: 2 }], // 10
+  [{ symbol: 0, length: 4 }], // 10
   [{ symbol: 1, length: 3 }, { symbol: 0, length: 3 }], // 5
 ];
 
+function shuffleArray(arr) {
+  for (let i = arr.length; i > 0; i -= 1) {
+    const j = Math.floor(Math.random() * i);
```

Comment:
This won't make random index.
I think you should use Math.floor(Math.random() * arr.length);
or you can use [Lodash's shuffle method](https://www.npmjs.com/package/lodash.shuffle).